internal/core/validation.go

```diff
@@ -25,20 +25,21 @@ import (
 	"strconv"
 	"strings"
 	"unicode"
+    "os"
```

Comment:
Please run `gofmt`. As a general recommendation, it's a good idea to have gofmt and/or goimports run automatically within your text editor ([this is what I have in Vim](https://github.com/majewsky/devenv/blob/master/toplevel/vimrc#L156-L157)), given that the Go ecosystem is pretty much fully standardized on gofmt's style.